docs/monitoring/containers.md

```diff
@@ -1,35 +1,266 @@
 title: Sematext Core Container Monitoring
-description: Monitor Docker, rkt and containerd containers with Sematext by installing simple Docker Certified Agents and using our interface that shows everything in one simple screen.
+description: Monitor Docker and containerd containers, container orchestration systems like kubernetes, swarm and nomad, or cloud container orchestration services EKS, ECS, AKS, GKE with Sematext by installing simple Docker Certified Agents and using our interface that shows everything in one simple screen.
 
-Monitor [Docker](https://www.docker.com/), rkt and containerd containers with Sematext by installing simple [Docker Certified Agents](https://hub.docker.com/_/sematext-agent-monitoring-and-logging) and using our interface that shows everything in one simple screen.
+Monitor [Docker](https://www.docker.com/) and containerd containers, container orchestration systems like kubernetes, swarm and nomad, or cloud container orchestration services EKS, ECS, AKS, GKE with Sematext by installing simple [Docker Certified Agents](https://hub.docker.com/_/sematext-agent-monitoring-and-logging) and using our interface that shows everything in one simple screen.
 
 ## Create a Sematext Monitoring App
-Creating a Sematext Monitoring App is as easy as choosing the Docker integration and giving the App a name.
+Creating a Sematext Monitoring App is as easy as choosing the Infra integration and giving the App a name.
 
 <video style="display:block; width:100%; height:auto;" controls autoplay>
-  <source src="https://cdn.sematext.com/videos/sematext-create-docker-app.mp4" type="video/mp4" />
+  <source src="https://cdn.sematext.com/videos/sematext-create-infra-app.mp4" type="video/mp4" />
 </video>
 
 ## Install the Sematext Agent
 Sematext can easily monitor your containers with the Sematext Agent. [Installing](../agents/sematext-agent/containers/installation) the Agent is as simple as running one command on each host.
 
 - [Docker](../agents/sematext-agent/containers/installation/#docker)
-- [Docker Compose](../agents/sematext-agent/containers/installation/docker-compose)
+- [Docker Compose](../agents/sematext-agent/containers/installation/#docker-compose)
 - [Docker Swarm](../agents/sematext-agent/containers/installation/#docker-swarm-enterprise)
```

Comment:
Removed on purpose?  I still see https://sematext.com/docs/agents/sematext-agent/containers/installation/#docker-swarm-enterprise

Reply:
I think it wasn't removed, this change was just about fixing the URL (without # it doesn't work).

Reply:
Aha.  The "-" confused me.